src/test/java/net/starlark/java/eval/Benchmarks.java

```diff
@@ -218,7 +218,7 @@ private static Benchmark run(String name, StarlarkFunction f, long budgetNanos)
     // Keep doubling the number of iterations until we exceed the deadline.
     // TODO(adonovan): opt: extrapolate and predict the number of iterations
     // in the remaining time budget, being wary of extrapolation error.
-    for (b.n = 1; b.time < budgetNanos; b.n <<= 1) {
```

Comment:
Let's change n to a long.

Reply:
On second thoughts that probably won't solve the problem by itself because it only takes twice as long to shift through 64 bits if the benchmark ignores b.n.

Let's assume that if a benchmark reaches overflow, it has made a mistake, and report an error. Even a null loop never gets close to overflow.